bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandlerFactory.java

```diff
@@ -233,18 +234,32 @@ public Thing createThing(ThingTypeUID thingTypeUID, Configuration configuration,
         if (thingType != null) {
             Thing thing = ThingFactory.createThing(thingType, thingUID, configuration, bridgeUID,
                     getConfigDescriptionRegistry());
+            Map<DefaultPropertyKey, String> staticProperties = getStaticProperties(thing);
+            if (staticProperties != null) {
+                for (DefaultPropertyKey staticProperty : staticProperties.keySet()) {
+                    thing.getConfiguration().put(staticProperty.name, staticProperties.get(staticProperty));
+                }
+            }
             return thing;
-        } else {
-            return null;
         }
+        return null;
     }
 
     protected ConfigDescriptionRegistry getConfigDescriptionRegistry() {
-        if (configDescritpionRegistryServiceTracker == null) {
+        if (configDescriptionRegistryServiceTracker == null) {
             throw new IllegalStateException(
                     "Config Description Registry has not been properly initialized. Did you forget to call super.activate()?");
         }
-        return configDescritpionRegistryServiceTracker.getService();
+        return configDescriptionRegistryServiceTracker.getService();
     }
 
+    /**
+     * Retrieves the {@link DefaultPropertyKey}s for the given {@link Thing}. Sub-classes should overwrite this operation
+     * in order to provide these kind of properties for the things to be created.
+     * 
+     * @param thing the {@link Thing} for which the {@link DefaultPropertyKey}s are to be retrieved
+     */
+    protected Map<DefaultPropertyKey, String> getStaticProperties(Thing thing) {
```

Comment:
This should be a Map<String, String> as static properties might not all have a default key.